app/routes/dev.tsx

```diff
@@ -12,8 +12,14 @@ import logger from '~/lib/logger.server';
 import { requireUser, requireUsername } from '~/session.server';
 import { addNotification } from '~/queues/notifications/notifications.server';
 import { addCertRequest } from '~/queues/certificate/certificate-flow.server';
+import { addDnsRequest } from '~/queues/dns/add-record-flow.server';
+import { updateDnsRequest } from '~/queues/dns/update-record-flow.server';
+import { deleteDnsRequest } from '~/queues/dns/delete-record-flow.server';
+import { createUserRecord, updateUserRecord } from '~/lib/record-db.server';
```

Comment:
Both create and update flow creator should handle the required DB changes. This should be abstracted away.